### PR TITLE
Retiarii exp launch

### PR DESCRIPTION
**Pull request by QuanluZhang**

### CODE REVIEW

File: ts/nni_manager/core/nnimanager.ts [hunk 1]
```diff
@@ -341,7 +341,6 @@ class NNIManager implements Manager {
                 }
             }
         }
-
```
Comment (ultmaster) [inline ts/nni_manager/core/nnimanager.ts:344]: This is a unrelated change?

File: nni/retiarii/experiment/pytorch.py [hunk 2]
```diff
@@ -131,7 +151,37 @@ def start(self, port: int = 8080, debug: bool = False) -> None:
         debug
             Whether to start in debug mode.
         """
-        super().start(port, debug)
+        atexit.register(self.stop)
+
+        self.id = management.generate_experiment_id()
+
+        if self.config.experiment_working_directory is not None:
```
Comment (ultmaster) [inline nni/retiarii/experiment/pytorch.py:158]: Is it a good idea to put these logics into Retiarii? Might as well create a new class in `nni.experiment`, called, for example, `_TunerCenteredExperiment`

**Review by ultmaster — APPROVED** [review]

File: nni/retiarii/experiment/pytorch.py (hunk 2 above)
Comment (liuzhe-lz) [reply nni/retiarii/experiment/pytorch.py:158]: I'll consider this a temporary implementation and should be refactored in upcoming release.

**Review by liuzhe-lz — APPROVED** [review]

File: nni/retiarii/experiment/pytorch.py (hunk 2 above)
Comment (QuanluZhang) [reply nni/retiarii/experiment/pytorch.py:158]: agree

File: ts/nni_manager/core/nnimanager.ts (hunk 1 above)
Comment (QuanluZhang) [reply ts/nni_manager/core/nnimanager.ts:344]: sorry, not related...
